Tests/MapboxNavigationTests/XCTestCase.swift

```diff
@@ -1,5 +1,6 @@
 import Foundation
 import XCTest
+import TestHelper
```

Comment:
do we really need this import?

Reply:
Probably not anymore, thanks for catching, I will check.